api/envoy/extensions/filters/http/aws_lambda/v3/aws_lambda.proto

```diff
@@ -64,4 +70,7 @@ message PerRouteConfig {
       "envoy.config.filter.http.aws_lambda.v2alpha.PerRouteConfig";
 
   Config invoke_config = 1;
+
+  // The human readable prefix to use when emitting stats.
+  string stat_prefix = 2;
```

Comment:
Is it possible to introduce `stat_prefix` updated on a separate PR? that can go first and then we can merge credentials_profile update

Reply:
I can do this first with the config refactor. and then add the profile change, no problem